api/updateContact.php

```diff
@@ -70,16 +70,6 @@ function checkIfEmailExists($uid, $email, $contactId)
     $email = $input['email'];              
 
     
-    if (checkIfPhoneExists($uid, $phoneNumber, $contactId)) {
-        http_response_code(409); 
-        echo json_encode(["message" => "Duplicate contact: The phone number already exists for another contact of this user."]);
-    }
-    
-    elseif (checkIfEmailExists($uid, $email, $contactId)) {
-        http_response_code(409); 
-        echo json_encode(["message" => "Duplicate contact: The email address already exists for another contact of this user."]);
-    } else {
-        
```

Comment:
wondering why you removed this?

Reply:
<img width="853" alt="image" src="https://github.com/user-attachments/assets/b364a158-88d6-4ea7-accd-cadc255917bd">
Check this photo for the current state of the app. 
When we display the list of contact, we HAVE the contact detail, so we should/must only let user update whatever available in their contactlist. 
Hence, the check list is not needed. In fact, the check is implemented wrong. 
Example: 

User: Update Contact Id 1, name Bob, phone: 12345

Backend api: update contact id 1, phone 12345 check ---  12345 already existed in the database, return error 
No update happens.

Reply:
This is auto-populated

<img width="1060" alt="image" src="https://github.com/user-attachments/assets/98b2b419-91f5-4b31-8677-73c74baa53b3">